superset-frontend/src/components/FacePile/FacePile.test.tsx

```diff
@@ -29,23 +28,37 @@ const users = [...new Array(10)].map((_, i) => ({
   id: i,
 }));
 
+jest.useFakeTimers();
```

Comment:
Please set it back to real timer after all specs are finished.

```suggestion
beforeEach(() => {
  jest.useFakeTimers();
});

afterEach(() => {
  jest.useRealTimers();
});
```